### PR TITLE
show when puller was completed successfully

### PR DESCRIPTION
**Pull request by grosser**
... also adding a clear DONE log line

@zendesk/compute

### CODE REVIEW

File: lib/secrets.rb [hunk 1]
```diff
@@ -278,4 +280,8 @@ def find_pod_ip_v4(pod_ip)
       first.
       ip_address
   end
+
+  def vault_timeout(&block)
+    Timeout.timeout 10, &block
```
Comment (adammw) [inline lib/secrets.rb]: You should be able to set open_timeout and read_timeout options on Vault::Client rather than using the timeout gem...

Comment (adammw) [reply lib/secrets.rb]: in fact it is already set...
```
      open_timeout: 3,
      read_timeout: 2
```

weird. why are we getting such long calls then?

**Comment by grosser** [comment]
we are not sure if this issue was still happening after we got rid of
kubeclient, was trying to avoid any issue by having this top-level timeout
:)

**Comment by adammw** [comment]
this is unrelated to kubeclient though..

**Comment by grosser** [comment]
the timeout reports we got never knew if it was a kubeclient or vault
issue, but both had there internal timeout settings
we can wait with this PR until we have another reported case to be sure
though

**Comment by grosser** [comment]
removed the timeouts until we need them ... keeping the logging change

**Pull request merged**